spec/update_quality_spec.rb

```diff
@@ -90,7 +91,7 @@
 
         before do
           # Verify that this is always true in the current context
-          award.expires_in.should == initial_expires_in
+          award.expires_in.should == initial_expires_in - 1
```

Comment:
The instructions did state that all awards aged and added no exception for 'Blue Distinction Plus' 
Assuming this is an error in the rspec file.  If instead this award does not age then I can add a fix for that into the `update_age` function within `update_quality.rb`.